viewer/cypress/shared.ts

```diff
@@ -80,3 +80,31 @@ export function downloadPng(selector: string, filename: string) {
     cy.wrap(path)
   })
 }
+
+export function injectAxe() {
+  //cy.injectAxe();
+  // cy.injectAxe is currently broken. https://github.com/component-driven/cypress-axe/issues/82
+
+  // Creating our own injection logic
+  cy.readFile('node_modules/axe-core/axe.min.js').then(source => {
+    return cy.window({ log: false }).then(window => {
+      window.eval(source)
+    })
+  })
+}
+
+export function checkAccessibility() {
+  cy.get('body').should('be.visible')
+  cy.checkA11y('body')

```

Comment:
Moet je body meegeven? We willen vaak de hele pagina checken namelijk

Reply:
Ja, specifiek in deze testen wordt alleen het component getest. Ik test hier niet de compliance van het cypress testbed ( In dit geval hoeft de pagina (https://github.com/PDOK/gokoala/blob/add-cypress-test-and-refactor/viewer/cypress/support/component-index.html)  niet compliance te zijn, omdat deze door cypress geleverd wordt.)